client/lib/reader-post-flux-adapter/index.js

```diff
@@ -29,9 +29,16 @@ const fluxPostAdapter = Component => {
 			const posts = map(
 				postKey.postIds,
 				postId => {
-					const post = FeedPostStore.get( { ...postKey, postId } )
-					if ( ! post ) {
-						fetchPost( { ...postKey, postId } );
+					const postKeyForPost = {};
```

Comment:
shouldn't matter, the posts in the _postsForBlogs store are the same instances as those in the _posts store.